### PR TITLE
Add specs for ezid metadata retrieval

### PR DESCRIPTION
**Pull request by fcd1**

### CODE REVIEW

**Comment by elo2112** [comment]
Other than my comment about "return nil" vs "return", this looks good!  I'm okay with merging once we hear back from @barmintor.

File: lib/hyacinth/ezid/hyacinth_metadata_retrieval.rb [hunk 1]
```diff
@@ -21,9 +21,10 @@ def initialize(digital_object_data_arg)
     # Following returns the title of an item. NOTE: if ever an item contains
     # multiple titles, it will only return the first one.
     def title
+      return unless @dfd.key? 'title'
```
Comment (barmintor) [inline lib/hyacinth/ezid/hyacinth_metadata_retrieval.rb]: I'm going to send a PR to this branch with some recommendations.

**Comment by elo2112** [comment]
Thanks for those additional commits, @barmintor.  Looks good.  Assigning ticket to @fcd1 for final approval, but I'm happy to see this merged into master at any time.

**Comment by barmintor** [comment]
Starting to pull YARD in more generally, while good in theory, is a big step. But I do think it would be wise to start incorporating some of it (especially in situations like `@return`) while we decide how much we want to use. We can talk about it this week.

**Pull request merged**